src/rtems_proxy/hybrid.py

```
@@ -0,0 +1,283 @@
+    """
+    nfs = GLOBALS.RTEMS_NFS_ROOT_PATH
+    runtime = GLOBALS.RUNTIME
+    ioc_root = GLOBALS.IOC_ORIGINAL_LOCATION
+
+    nfs_runtime = Path(nfs) / "runtime"
+    nfs_ioc = Path(nfs) / "ioc"
+    nfs_runtime.mkdir(parents=True, exist_ok=True)
+    nfs_ioc.mkdir(parents=True, exist_ok=True)
+
+    # runtime/: startup script and the expanded database
+    subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)